Tensile/KernelWriterAssembly.py

```diff
@@ -10951,7 +10965,10 @@ def globalWriteBatch(self, kernel, ss, batchIdx, applyAlpha, beta, edge, atomic,
 
       if beta:
         kStr += addrCalc.emitLdChange(kernel, ss, 'C', edge, beta, mask, (elementIdx == 0), tmpVgpr, addr, addrC)
-        kStr += self.readCInput(kernel, ss, addrCalc, vc0, data, gwvw, addr, tmpS01)
+        if kernel["GroupLoadStore"]:
+          loadBetaCode += self.readCInput(kernel, ss, addrCalc, vc0, data, gwvw, addr, tmpS01)
```

Comment:
LoadBetaCode would refer to addrCalc.addrVgpr which is calculated by emitLdChang for C matrix.